config/config.yml

```diff
@@ -1,5 +1,5 @@
-broker: pyamqp://guest@localhost//
-backend: rpc://
+broker: redis://guest@localhost//
+backend: redis://
```

Comment:
Am I right in thinking leaving the rest of this url blank is actually equivalent to `redis://guest@localhost/0/` because those are the defaults?